dev/com.ibm.websphere.appserver.features/visibility/auto/io.openliberty.cdi4.1-expressionLanguage6.0.feature

```diff
@@ -3,8 +3,8 @@ symbolicName=io.openliberty.cdi4.1-expressionLanguage6.0
 Manifest-Version: 1.0
 IBM-API-Package: jakarta.enterprise.inject.spi.el;  type="spec"
 IBM-Provision-Capability: \
-  osgi.identity; filter:="(&(type=osgi.subsystem.feature)(osgi.identity=io.openliberty.cdi-4.1.feature))", \
+  osgi.identity; filter:="(&(type=osgi.subsystem.feature)(osgi.identity=io.openliberty.cdi-4.1))", \
   osgi.identity; filter:="(&(type=osgi.subsystem.feature)(osgi.identity=io.openliberty.expressionLanguage-6.0))"
 IBM-Install-Policy: when-satisfied
```

Comment:
Will a bundle eventually be in this auto feature?  If not, you might as well expose the API in the cdi-4.1 feature and not need this feature at all.

Reply:
Not sure. However, the classes in this package reference the EL API which is exposed by the EL feature, so the user can't use them unless they have both CDI and EL enabled.